.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,589 @@
+		"orange-6": #802500,
+		"orange-7": #682000,
+		"orange-8": #4f1900,
+		"orange-9": #371200,
+		"orange-10": #1f0b00,
+		"green-1": #80d971,
+		"green-2": #57c244,
+		"green-3": #34ab1f,
+		"green-4": #189402,
+		"green-5": #137d00,
+		"green-6": #0f6600,
+		"green-7": #0c4f00,
+		"green-8": #083800,
+		"green-9": #052100,
+		"green-10": #020a00,
+		"neutral-1": #e6efff,
+		"neutral-2": #c3cfe3,
+		"neutral-3": #a3b0c6,
+		"neutral-4": #8593aa,
+		"neutral-5": #69778e,
+		"neutral-6": #4f5c71,
+		"neutral-7": #384355,
+		"neutral-8": #232b39,
+		"neutral-9": #10151c,
+		"neutral-10": #000000,
+		"spacing-1": 0.25rem,
+		// 4px
+		"spacing-2": 0.5rem,
+		// 8px
+		"spacing-3": 1rem,
+		// 16px
+		"spacing-4": 1.5rem,
+		// 24px
+		"spacing-5": 2rem,
+		// 32px
+		"spacing-6": 2.5rem,
+		// 40px
+		"spacing-7": 3rem,
+		// 48px
+		"spacing-8": 3.5rem,
+		// 56px
+		"spacing-9": 4rem,
+		// 64px
+		"spacing-10": 4.5rem,
+		// 72px
+		"spacing-11": 5rem,
+		// 80px
+		"spacing-12": 5.5rem,
+		// 88px
+		"spacing-13": 6rem,
+		// 96px
+		"spacing-14": 6.5rem,
+		// 104px
+		"spacing-15": 7rem,
+		// 112px
+		"spacing-16": 7.5rem,
+		// 120px
+		"font-weight-1": 100,
+		"font-weight-2": 200,
+		"font-weight-3": 300,
+		"font-weight-4": 400,
+		"font-weight-5": 500,
+		"font-weight-6": 600,
+		"font-weight-7": 700,
+		"font-weight-8": 800,
+		"font-weight-9": 900,
+		"font-size-1": 0.625rem,
+		// 10px
+		"font-size-2": 0.75rem,
+		// 12px
+		"font-size-3": 0.875rem,
+		// 14px
+		"font-size-4": 1rem,
+		// 16px
+		"font-size-5": 1.125rem,
+		// 18px
+		"font-size-6": 1.1875rem,
+		// 19px
+		"font-size-7": 1.25rem,
+		// 20px
+		"font-size-8": 1.375rem,
+		// 22px
+		"font-size-9": 1.5rem,
+		// 24px
+		"font-size-10": 1.75rem,
+		// 28px
+		"font-size-11": 2rem,
+		// 32px
+		"font-size-12": 2.25rem,
+		// 36px
+		"font-size-13": 2.56rem,
+		// 41px
+		"font-size-14": 3rem,
+		// 48px
+		"font-size-15": 3.25rem,
+		// 52px
+		"line-height-1": 0.875rem,
+		// 14px
+		"line-height-2": 1rem,
+		// 16px
+		"line-height-3": 1.25rem,
+		// 20px
+		"line-height-4": 1.5rem,
+		// 24px
+		"line-height-5": 2rem,
+		// 32px
+		"line-height-6": 2.125rem,
+		// 34px
+		"line-height-7": 2.5rem,
+		// 40px
+		"line-height-8": 3rem,
+		// 48px
+		"line-height-9": 3.5rem,
+		// 56px
+		"line-height-10": 3.875rem,
+		// 62px
+		"box-shadow-1": 0px 0px 16px rgba(25, 25, 25, 0.4),
+		"z-index-1": 1,
+		"z-index-2": 10,
+		"z-index-3": 50,
+		"z-index-4": 100,
+		"z-index-5": 500,
+		"border-radius-1": 0.25em,
+	),
+	$alias: (
+		"layout-max-width": 1440,
+		"layout-content-max-width": 1216,
+		//
+		// --layout-content-scale-width identifies the inner document width for header and content
+		// alignment.  It accounts for a minimum width that has (--global-spacing-5 / 2) margins
+		//
+		"layout-content-scale-width":
+			min(
+				calc(100vw - var(--global-spacing-5)),
+				calc(var(--layout-content-max-width) / var(--layout-max-width) * 100%)
+			),
```

Comment:
These types of values are going to be very interesting to consider in how we approach them in Theme Settings 🤔

Reply:
I share this concern. This came in with the tokens file from the `arc-themes-feature-pack` repository.
https://github.com/WPMedia/arc-themes-feature-pack/blob/commerce/site-styles/arc-commerce/_index.scss
It was that file to maintain source of truth and prevent fragmentation in token usage.

Reply:
Actually - lemme just tag 'em.

@jgrosspietsch @annasherman ^ Peep this 👀